semantic_conventions.md

```diff
@@ -52,9 +53,18 @@ Every Span log has a specific timestamp (which must fall between the start and f
 The following Span tags combine to model RPCs:
 
 - `span.kind`: either `"client"` or `"server"`. It is important to provide this tag **at Span start time**, as it may affect internal ID generation.
-- `error`: whether the RPC ended in an error
 - `peer.hostname`, `peer.ipv4`, `peer.ipv6`, `peer.port`, `peer.service`: optional tags that describe the RPC peer (often in ways it cannot assess internally)
 
+### Messaging
```

Comment:
As you say, it doesn't express mq, which I think would cause confusion. So how about a compromise, "Message Bus", as suggested by @devinsba, but `messagebus` (or `message_bus.`) prefix, as `mb` is not a well known abbreviation?